cxx-qt/src/lib.rs

```diff
@@ -37,9 +37,6 @@ pub fn bridge(args: TokenStream, input: TokenStream) -> TokenStream {
     // this triggers a compile failure if the tokens fail to parse.
     let mut module = parse_macro_input!(input as ItemMod);
 
-    // Macros do not typically need to do anything with their own attribute name,
-    // so rustc does not include that in the `args` or `input` TokenStreams.
-    //
```

Comment:
Was deleting this an accident when resolving conflicts?

Reply:
Oops, yup, reverted.